ccd-definition/AuthorisationCaseField/CareSupervision/court-admin.json

```diff
@@ -2449,6 +2449,13 @@
     "UserRole": "caseworker-publiclaw-courtadmin",
     "CRUD": "R"
   },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseFieldID": "courtBundleListV2",
+    "UserRole": "caseworker-publiclaw-courtadmin",
+    "CRUD": "RU"
```

Comment:
This was left from Kian's changes previously - not sure why it would change as I think they only need to be able to read the list. Can probably revert to R.